Samples/azure-storage/Azure.Ruby/generated/azure_storage/storage_management_client.rb

```diff
@@ -116,5 +117,17 @@ def make_request_async(method, path, options = {})
       super(request_url, method, path, options)
     end
 
+
+    private
+    #
+    # Adds telemetry information.
+    #
+    def add_telemetry
+        sdk_information = 'azure_storage'
+        if defined? Petstore::VERSION
```

Comment:
Separately from this specific change, it looks like the AzureStorage sample is being generated with Petstore name (check the module name at the top), we may need to fix that either here or in a different issue).